.travis.yml

```diff
@@ -1,25 +1,52 @@
 language: python
 
+sudo: false
 
 env:
-    - CONDA="python=2.7"
-    - CONDA="python=3.4"
-    - CONDA="python=3.5"
-    - CONDA="python=3.6"
+  global:
+    - secure: "DdUE82IkNYo317595sy4H6z3GQoaVUTAAUDOOoJATa199ERxSQy8VlsMKELdRTrBEt76kSBJdjjJ7YSiOVSMt9qao9XWOWYjtfdX0KpbkmfiHMH6fmQnckNndMbPY8wa42fFusIWOxZL/jCay1Gq0GppjoHdSoaLGIk3hLkYE1U="
+
+addons:
+  apt:
+    packages: octave
+
+matrix:
+  fast_finish: true
+  include:
+  - python: 2.7
+    env: TEST_TARGET=default NUMPY=1.11
+  - python: 2.7
+    env: TEST_TARGET=default NUMPY=1.12
+  - python: 3.6
+    env: TEST_TARGET=default NUMPY=1.11
+  - python: 3.6
+    env: TEST_TARGET=default NUMPY=1.12
```

Comment:
I don't think we should test Python 3.4 and 3.5. Testing multiple NumPy version is more important here.